components/style/restyle_hints.rs

```diff
@@ -847,12 +1035,12 @@ impl DependencySet {
                 return true;
             }
 
-            if hint.contains(dep.hint) {
+            if hint.contains(&dep.hint) {
                 trace!(" > hint was already there");
                 return true;
             }
 
-            // We can ignore the selector flags, since they would have already
+            // We can ignore the selector replacements, since they would have already
```

Comment:
This comment is talking about the slow selector flags, so I don't think it should be changed.